src/modules/networks/NetworkConfig.ts

```diff
@@ -9,9 +9,10 @@ interface NetworkAsset {
 }
 
 type NetworkSpecification = {
-  standard: 'eip155';
+  standard: 'eip155' | 'solana';
   specification: {
-    eip155: null | {
+    solana?: null | object;
```

Comment:
can we specify this better?

Reply:
I did it like this because backend said that the current value isn't final and we don't need to use it now anyway
Wdyt?